recipes-bsp/sgx/sgx-sdk-cross_2.18.1.bb

```diff
@@ -40,6 +40,17 @@ do_install () {
         cd ${D}${sgxdirprefix}${sgxsdkpath}-cross/lib64
         ln -sf libsgx_urts.so libsgx_urts.so.2
     fi
+    # Fix for qa check buildpaths that contains reference to TMPDIR
```

Comment:
Please add more description for below change. It is not clear which TMPDIR is being referenced here.

Reply:
> Please add more description for below change. It is not clear which TMPDIR is being referenced here.

Additional comment added with TMPDIR path for reference.